data/yaser-alazm.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Yaser Alazm",
+  "type": "personal",
+  "bio": "Fullstack Javascript Engineer",
+  "avatar": "https://github.com/yaser-alazm.png",
```

Comment:
Hi @yaser-alazm, thank you for contributing  :tada: 
You can delete this line, it is not required anymore :+1: 
```Suggestion
```

Reply:
thanks @khushalarora 
just updated 👍